partiql-types/src/main/kotlin/org/partiql/errors/ErrorCode.kt

```diff
@@ -12,37 +12,34 @@
  *  language governing permissions and limitations under the License.
  */
 
-package org.partiql.lang.errors
-
-import org.partiql.lang.eval.ExprValueType
-import org.partiql.lang.syntax.impl.DateTimePart
+package org.partiql.errors
 
 /** Property Set constants used in [ErrorCode] */
 private val LOCATION = setOf(Property.LINE_NUMBER, Property.COLUMN_NUMBER)
 private val TOKEN_INFO = setOf(Property.TOKEN_DESCRIPTION, Property.TOKEN_VALUE)
 private val LOC_TOKEN = LOCATION + (TOKEN_INFO)
 private val LOC_TOKEN_STR = LOCATION + (setOf(Property.TOKEN_STRING))
 
-enum class ErrorBehaviorInPermissiveMode {
+public enum class ErrorBehaviorInPermissiveMode {
     THROW_EXCEPTION, RETURN_MISSING
 }
 
-internal const val UNBOUND_QUOTED_IDENTIFIER_HINT =
+public const val UNBOUND_QUOTED_IDENTIFIER_HINT: String =
```

Comment:
Does this need to be `public`?

Reply:
It is used in two places, but honestly need not exist because it's just a debug tring https://github.com/search?q=repo%3Apartiql%2Fpartiql-lang-kotlin%20UNBOUND_QUOTED_IDENTIFIER_HINT&type=code

We can remove this (which also relates to Vladimir's concern)